service/src/test/java/com/commercetools/main/BasicPaymentRequestWorkflow.java

```diff
@@ -43,11 +62,11 @@ public void setUp() throws URISyntaxException {
         Main.main(new String[0]);
     }
 
-    //TODO we need to create ctp resources and try to simulate handle payment call
-    // to ensure that payone-integration service will work as expected
     @Test
-    public void makeBankTransferPayoneRequest() throws IOException {
-        HttpResponse httpResponse = executeGetRequest(format(URL_PATTERN, 8080));
-        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(200);
+    public void verifyCreditCardTransferPayoneRequestIsSuccess() throws Exception {
+        final String paymentId = createPayment("CREDIT_CARD", "CHARGE", "40", "EUR");
+
+        HttpResponse httpResponse = executeGetRequest(format(URL_HANDLE_PAYMENT+paymentId, DEFAULT_PORT));
+        assertThat(httpResponse.getStatusLine().getStatusCode()).isIn(HttpStatus.SC_ACCEPTED, HttpStatus.SC_OK);
```

Comment:
Maybe you can assert more fields in the payment? Just fetch the payment object by ID and assert?

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/392/commits/f04cec2a5e760744ac66b6102ca247dc52629d1c